### DISCUSSION TITLE
Does opensource `emqx` supports bridge?

### DISCUSSION THREAD

**Discussion by Bhautik0110** [emqx/emqx#9442] [Q&A] [answered]
### Question
- Does open-source version of `emqx` supports the bridge feature?

**Comment by lafirest** [comment, marked as answer]
@Bhautik0110 
Yes,  You can find it [here](https://www.emqx.io/docs/en/v5.0/data-integration/data-bridges.html)

**Reply by TinkeringAround** [reply]
I tried to follow the instructions, very basic at first. However, I always get the following error when testing the connection.
A corresponding rule does match all incoming request, but is only queuing them. Not even try to send them to the webhook I try to setup here. Any ideas how to approach this?

<img width="436" alt="emqx-error" src="https://user-images.githubusercontent.com/33089847/212906888-04cc5db4-7998-4cae-b1cb-967d0af88970.png">
<img width="1020" alt="1" src="https://user-images.githubusercontent.com/33089847/212907252-cea4dfb1-e8a8-49aa-8d88-fbceee2d418a.png">
<img width="944" alt="2" src="https://user-images.githubusercontent.com/33089847/212907261-c52acab5-b3f5-4023-a111-8dc3ed4c5117.png">


Using this k8s helm deployment config for emqx version. 5.0.14:
https://github.com/emqx/emqx/blob/master/deploy/charts/emqx/values.yaml

**Reply by lafirest** [reply]
@TinkeringAround 
It seems there is some network connection problem with your config or environment from your words.
But the `404` warning is also strange.
I think this may be another problem, you can pull a new issue or discussion, and with detailed logs.
